app/helpers/time_summary_helper.rb

```diff
@@ -59,7 +59,11 @@ def time_summary_url time_summary
   end
 
   def time_summary_enabled_edit item, date
-    date >= 1.week.ago
+    date >= 3.week.ago
```

Comment:
```suggestion
    date >= 3.weeks.ago
```
Están las dos versiones, son un alias, pero cuando el número no es 1 se usa esta así se lee bien.